rules/framework.bzl

```diff
@@ -921,7 +922,7 @@ def _bundle_dynamic_framework(ctx, is_extension_safe, avoid_deps):
         ] + processor_result.providers,
     )
 
-def _bundle_static_framework(ctx, is_extension_safe, current_apple_platform, outputs):
+def _bundle_static_framework(ctx, is_extension_safe, current_apple_platform, outputs, product_type):
```

Comment:
Similar comment as above, do we have a need to allow the function `_bundle_static_framework` to accept other product types than a static framework?

Reply:
We don't have to, this was just for better readability and to be explicit about the type. Happy to remove and just let the default to be used here (which is static).

Reply:
https://github.com/bazel-ios/rules_ios/pull/741/commits/c209ea96a3ccd838c37a94cfc42a77345bb7a20f